docs/processors/getting-started.md

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Getting Started with Processors'
+slug: 'getting-started'
+sidebar_label: "Getting Started"
+---
+
+A processor is a component that operates on a single record that flows through a pipeline. It can either change the record
+(i.e. **transform** it) or **filter** it out based on some criteria. Since they are part of pipelines, making yourself
+familiar with [pipeline semantics](/docs/introduction/pipeline-semantics) is highly recommended.
+
+![Pipeline](/images/pipeline_example.svg)
+
+Processors are **optional** components in a pipeline, i.e. a pipeline can be started without them. They are always attached
+to a single parent, which can be either a connector or a pipeline. With that, we can say that we have the following types
+of processors:
+1. **Source processors**: these processors only receive messages originating at a specific source connector. Source
+   processors are created by specifying the corresponding source connector as the parent entity.
+2. **Pipeline processors**: these processors receive all messages that flow through the pipeline, regardless of the
+   source or destination. Pipeline processors are created by specifying the pipeline as the parent entity.
+3. **Destination processors**: these processors receive only messages that are meant to be sent to a specific
+   destination connector. Destination processors are created by specifying the corresponding destination connector as the
+   parent entity.
+
+Given that every processor can have one (and only one) parent, processors cannot be shared. In case the same processing
+needs to happen for different sources or destinations, you have two options:
+1. If records from all sources (or all destinations) need to be processed in the same way, then you can create
+   a pipeline processor
+2. If records from some, but not all, sources (or destinations) need to be processed in the same way, then you need to
+   create multiple processors (one for each source or destination) and configure them in the same way.
+
+## Adding and configuring a processor
+
+Processors are created through the `/processors` endpoint. Here's an example:
+
+```json lines
+POST /v1/processors
+{
+    // name of the processor in Conduit
+    // note that this is NOT a user-defined name for this processor
+    "name": "extractfieldpayload",
```

Comment:
This would now be `type`? It looks the processors' readme in Conduit needs an update too.:)